### ISSUE TITLE
Kebap Cases for KITs 

### ISSUE THREAD

**Issue by danielmiehle** [eclipse-tractusx/eclipse-tractusx.github.io#719]
### Acceptance Criteria

- [ ] Requirement is added to the KIT TRG
- [ ] All KITs use kebab cases in their urls

### Additional Information

Kebab case -- or kebab-case -- is a programming variable naming convention where a developer replaces the spaces between words with a dash.

Bad example:
https://eclipse-tractusx.github.io/docs-kits/kits/Data%20Chain%20Kit/Adoption%20View%20Data%20Chain%20Kit

Good example:
https://eclipse-tractusx.github.io/docs-kits/kits/tractusx-edc/CHANGELOG

**Comment by mhellmeier** [comment]
Please remember that the current URLs are used on different pages. Changing them can cause 404 errors.

**Comment by SebastianBezold** [comment]
> Please remember that the current URLs are used on different pages. Changing them can cause 404 errors.

Hi @mhellmeier, 

do you mean, that pages like KITs for example do reference other pages and the links need to be fixed?
If so, I think we should come up with some guidance on how to link. I would prefer linking via filename instead of relative URLs. URLs should only be used for external content.
See [Docusaurus docs](https://docusaurus.io/docs/markdown-features/links). Especially have a look at the "benefits" section at the bottom. 
One pitfall could be the "included by the same plugin," section mentioned in the docs. Not sure if they refer to the `content-docs` plugin or dedicated instances of this plugin. We are using it in multiple configs. Default for "Developer Hub" and extra ones for "KITs" and "Community"

**Comment by mhellmeier** [comment]
> do you mean, that pages like KITs for example do reference other pages and the links need to be fixed?

Exactly! There are currently files or folders with spaces in it. Even if the links are relative, changing them would break different things.
